src/app_service/service/chat/service.nim

```diff
@@ -343,8 +343,28 @@ QtObject:
     result = ""
     try:
       var images = Json.decode(imagePathsJson, seq[string])
+      let base64JPGPrefix = "data:image/jpeg;base64,"
 
       for imagePath in images.mitems:
+
+        if imagePath.startsWith(base64JPGPrefix):
+          # If an image was copied to clipboard we're receiving it from there
+          # as base64 encoded string. The base64 string will always have JPG data
+          # because image_resizer (a few lines below) will always generate jpgs
+          # (which needs to be fixed as well). 
+          #
+          # We save the image data to a tmp location because image_resizer expects
```

Comment:
Maybe it is a good idea to make some cleanups and remove tmp files in the end.

Reply:
QTemporaryFile... oh wait, we can't use all the Qt stuff :/

Reply:
Nah, I'd say it's fine

Reply:
Turns out they were removed as the references of paths has changed after processing.
I've made this now explicit